09-student_guide.Rmd

```diff
@@ -0,0 +1,277 @@
+
+install.packages('phangorn')
+```
+
+Once you've downloaded a package, it will be saved on your computer (or, in the case of AnVIL, on your persistent disk space) so that you don't have to download it again. Anytime you want to use the set of commands that are stored in a particular package, you'll tell R to open the package with the "library" command. 
+
+Let's open both packages now:
+
+```{r, message = FALSE, warning = FALSE}
+library(ape)
+library(phangorn)
+```
+
+You can verify that both packages have been loaded by looking at the Packages tab in the lower left-hand window of the RStudio interface. Packages that have been loaded are checked. You can search specifically for each package, or scroll down the entire list.
+
+<img src="resources/images/09-check_packages.png" title="How to check for loaded packages" alt="How to check for loaded packages" style="display: block; margin: auto;" />
+
+
+## Exercise Two: Examining fasta files in RStudio
+
+Now we need to retrieve the data. We'll start by loading a type of data file called a fasta file. The fasta format is a common way to store sequences (either DNA or protein). Each sample in a fasta file has two sections. The sample ID and other descriptive information is on the first line (the description line). This line begins with either a > or a ;. The sample sequence is on the line immediately after the description. The sequence is written in standard IUCAC codes for either nucleic acids (for DNA sequence) or amino acids (for protein sequence). The sequence can also include unknown bases or gaps.
+
+The fasta file we're loading first contains the aligned sequences for the spike protein of 5 SARS-CoV-2 samples. This is what the top of the file looks like in a text editor:
+
+<img src="resources/images/09-fasta.png" title="Examples of a fasta file format" alt="An example of the first few lines of a fasta file" style="display: block; margin: auto;" />
+
+We can load this file into RStudio using the `read.FASTA` command and save it as the object "spike.small".
+
+```{r, message = FALSE, warning = FALSE}
+spike.fasta <- read.FASTA("sars_spike_protein.fasta")
+```
+
+After we've created an object in RStudio, we can get information about the object by typing the object's name.
+
+```{r, message = FALSE, warning = FALSE}
+spike.fasta
+```
+
+Here we can see a summary of what this object contains, as well as how long the sequences are and the sequence names. 
+
+Notice that RStudio has saved the information in the fasta file as binary data. This means the sequence information has been converted from "ATCG" into something easier for RStudio to work with, but harder for humans to interpret.
+
+The `phangorn` package uses a special data format called `phyDat`, which is derived from the fasta format. A `phyDat` object provides some additional information about the samples we upload. 
+
+```{r, message = FALSE, warning = FALSE}
+spike.phydat <- read.phyDat("sars_spike_protein.fasta", format = "fasta")
```

Comment:
will this work even with a cloned workspace?

Reply:
I'm not sure I understand the question.